_episodes/02-variables.md

```diff
@@ -47,15 +48,34 @@ first_name = 'Ahmed'
 *   To add a string to the printout, wrap the string in single quotations.
 *   The values passed to the function are called 'arguments'
 
+~~~
+print(first_name)
+~~~
+{: .python}
+~~~
+Ahmed
+~~~
+{:.output}
+
+Python's `print` function can also display multiple items at once.
+
 ~~~
 print(first_name, 'is', age, 'years old')
+print first_name, 'has', eye_color, 'eyes'
 ~~~
 {: .python}
 ~~~
-Ahmed is 42 years old
+('Ahmed', 'is', 42, 'years old')
+Ahmed has green eyes
 ~~~
 {: .output}
 
+Note in the first case without parenthesis we get a different output format than
+in the second case. This is just a difference in the way that our version of
+iPython prints information. Both are ok, but the first version (with
+parenthesis) will work on any version of Python, so we'll continue to teach that
+throughout the workshop today.
+
 *   `print` automatically puts a single space between items to separate them.
```

Comment:
This statement is not true now for python 2.7

Reply:
I think this is false when printing with parenthesis in 2.7 as we're going to do for the workshop, but when printing in 2.7 w/o parenthesis or in 3 with parenthesis I think a space is still added. 

Either way, I'll clarify this here for the learners.